sdcm/sct_events.py

```diff
@@ -462,6 +462,35 @@ def __str__(self):
             super(SpotTerminationEvent, self).__str__(), self)
 
 
+class PrometheusAlertEvent(SctEvent):  # pylint: disable=too-many-instance-attributes
+    def __init__(self, event_type:str, annotations: dict = None, startsAt=None, endsAt=None, updatedAt=None, fingerprint: str = None,
```

Comment:
this is not PEP 8, we use CamelCase only for class names

Reply:
problem that code fill these variables from the data it gets from openapi endpoint.
It is done to not have excess mapping code.

Reply:
`PrometheusAlert` will get the raw alert dict and then map the values to the instance attribute
```
class PrometheusAlert:
	def __init__(self, raw_alert):
		self.updated_at = raw_alert["updatedAt"]
		self.generator_url = raw_alert["generatorURL"]
                ...
```

Reply:
fixed